Imagine/Data/DataManager.php

```diff
@@ -149,8 +149,8 @@ public function find($filter, $path)
             throw new LogicException(sprintf('The mime type of image %s was not guessed.', $path));
         }
 
-        if (0 !== mb_strpos($binary->getMimeType(), 'image/')) {
-            throw new LogicException(sprintf('The mime type of image %s must be image/xxx got %s.', $path, $binary->getMimeType()));
+        if (0 !== mb_strpos($binary->getMimeType(), 'image/') && $binary->getMimeType() <> 'application/pdf') {
```

Comment:
```suggestion
        if (0 !== mb_strpos($binary->getMimeType(), 'image/') && $binary->getMimeType() !== 'application/pdf') {
```

while `<>` seems to work too, i think !== is much more commonly used. is there a particular reason to use `<>` here?

Reply:
As there's no difference and absolutely no technical reason to use one over the other. For consistency we can switch to most commonly used operator. 
Thanks